test/e2e/apiserver.go

```diff
@@ -262,7 +262,7 @@ var _ = framework.KubeDescribe("API Server webhook tests for pods (update path)
 		image = "skipper"
 		tag = "v0.13.97" // this image tag is compliant as well
 
-		pod = createImagePolicyWebhookTestPod(pod.Name, ns, image, tag, podname)
+		pod = updateImagePolicyWebhookTestPod(pod.Name, ns, image, tag, podname)
 		_, err = cs.CoreV1().Pods(ns).Update(context.TODO(), pod, metav1.UpdateOptions{})
```

Comment:
This won't work because of the versioning. You need to update the previous pod object you get from the API, or send a patch instead.

Reply:
🤦   That makes sense